package.json

```diff
@@ -13,7 +13,7 @@
   "dependencies": {
     "@docusaurus/core": "^2.0.0-beta.20",
     "@docusaurus/preset-classic": "^2.0.0-beta.20",
-    "@govtechsg/tradetrust-ui-components": "^2.18.0",
+    "@govtechsg/tradetrust-ui-components": "^2.18.6",
     "@govtechsg/tradetrust-utils": "^1.2.1",
```

Comment:
`2.18.7`?

Reply:
yup, cos docusaurus not using our nav bar so it wont really affect it.